CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 ## CHANGE LOG.
 
+### September 20, 2022
+* [CleverTap Android SDK v4.6.3](https://github.com/CleverTap/clevertap-android-sdk/blob/master/docs/CTCORECHANGELOG.md)
+
 ### September 13, 2022
 * [CleverTap Android SDK v4.6.2](https://github.com/CleverTap/clevertap-android-sdk/blob/master/docs/CTCORECHANGELOG.md)
+
```

Comment:
@root-ansh-ct Please mention in the Sep 13 and Sep 6 sections, please mention that if they are using App Inbox without tabs, then they need to use v4.6.3